### DISCUSSION TITLE
log out when callback

### DISCUSSION THREAD

**Discussion by e3ang** [evryn/laravel-toman#27] [Q&A] [answered]
hi 
thanks for package
when callback from api idpay
get 419 expire page 
i added url call back to VerifyCsrfToken.php
also when back
normaly rout to login page and log out from account 
how can i fix that

**Comment by AmirrezaNasiri** [comment, marked as answer]
Hi,
It's probably something with the authentication and not the package. Make sure that CSRF doesn't apply to the callback route and authentication is not needed for the route, since it'll cause the issue you mentioned.

**Reply by AmirrezaNasiri** [reply]
I'm marking the discussion as resolved. Let me know if you're still facing the issue.
